lib/alloc_trace.cpp

```diff
@@ -75,7 +75,8 @@ std::ostream &operator<<(std::ostream &out, const AllocTrace &at) {
     std::sort(sorted.begin(), sorted.end(), [](auto &a, auto &b) { return a.first > b.first; });
 #if HAVE_LIBBACKTRACE
     if (!global_backtrace_state)
-        global_backtrace_state = backtrace_create_state(exename(), 1, bt_error, &out);
+        global_backtrace_state =
+            backtrace_create_state(getExecutablePath().c_str(), 1, bt_error, &out);
```

Comment:
Per documentation:
>  Create state information for the backtrace routines.  This must be
called before any of the other routines, and its return value must
be passed to all of the other routines.  FILENAME is the path name
of the executable file; if it is NULL the library will try
system-specific path names.  If not NULL, FILENAME must point to a
permanent buffer.

So, we'd need to ensure `getExecutablePath` output is permanent. Probably the easiest (though a bit dirty) way is to roundtrip via `cstring` here...